File: week2/day2/draw-square.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %   %
# %   %
# %   %
# %   %
# %%%%%
#
# The square should have as many lines as the number was

x = int(input("Please enter a number for the square: "))
a = 0

print((x-1) * "%")

while a < (x - 2):
    a += 1
    print("%" + (x - 3) * " " + "%")

print((x-1) * "%")

File: week8/day2/draw-square.js
'use strict';

var lineCount = 10;

// Write a program that draws a
// square like this:
//
//
// %%%%%
// %   %
// %   %
// %   %
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

var block = '%'
var space = ' '

console.log(block.repeat((lineCount-1)));
for (var i = 0; i < (lineCount - 2); i++) {
    console.log(block + space.repeat(lineCount - 3) + block);
}
console.log(block.repeat((lineCount-1)));
